src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,48 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+    Animal.addAlive(this);
+  }
+
+  static alive = [];
+
+  static addAlive(animal) {
+    Animal.alive.push(animal);
+  }
+  static removeDead(animal) {
+    Animal.alive = Animal.alive.filter((aliveAnimal) => aliveAnimal !== animal);
```

Comment:
The `removeDead` method correctly filters out the dead animal, but make sure that this method is called whenever an animal's health drops to zero or below to keep the `alive` list accurate.